duo_client/admin.py

```diff
@@ -2833,7 +2833,7 @@ def delete_bypass_code_by_id(self, bypass_code_id):
             Notes:
                 Raises RuntimeError on error.
         """
-        registration_id = \
```

Comment:
It seems like this function would not have worked at all before you made this change. Is that true?

Reply:
Not necessarily. 
The path is still using bypass_code_id, I assume this might cause some python2v3 bug since it was not proper parsed with six

Reply:
I think if would fail only if the bypass code id were not url-safe?  the bug means we were just using the bypass code id passed into the method and not the one that was url encoded?

Reply:
gotcha, I think this is fine to merge. I just wanted to make sure I understand exactly what our fix was